klaytnetl/jobs/exporters/blocks_and_transactions_item_exporter.py

```diff
@@ -36,8 +36,8 @@
     "extra_data",
     # 'gas_limit',  # Does not supported by klay_getBlockWithConsensusInfoByNumber
     "gas_used",
-    "timestamp",
-    "unix_timestamp",
+    "block_timestamp",
+    "block_unix_timestamp",
```

Comment:
Do you know the difference between the fields `block_timestamp` and `block_unix_timestamp`?

Reply:
block_timestamp will have timestamp datatype (iso-format) where as block_unix_timestamp will have float datatype (epoch).